markdown/summary.Rmd

```diff
@@ -19,30 +42,31 @@ contact <- data.frame(contact1, contact2, contact3)
+This release by the Information Services Division provides information on the percentage of time spent at home or in a community setting in the last six months of life. Data are presented by financial year from `r min(basefile$fy)` to `r max(basefile$fy)`, by NHS Board, Health and Social Care Partnership of residence and for Scotland. 
 
-Note that hospital data and National Records of Scotland deaths data are provisional for 2018/19. This publication will be updated in October 2019 with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
+Note that hospital data and National Records of Scotland deaths data are provisional for `r max(basefile$fy)`. This publication will be updated in October `r year(end_date)` with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
 
 
 ### Main Points
 
-* In 2018/19, there were 53,168 deaths in Scotland excluding those where an external cause such as unintentional injury was recorded.  For these individuals, 89.2% of their last six months of life was spent either at home or in a community setting, with the remaining 10.8% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
+* In `r latest$fy`, there were `r latest$deaths` deaths in Scotland excluding those where an external cause of death was recorded (including falls).  For these individuals, `r latest$qom`% of their last six months of life was spent either at home or in a community setting, with the remaining `r latest$qom_hosp`% spent in hospital. This is equivalent to an average of `r round_half_up(182.5 * (latest$qom_hosp / 100))` days in hospital in the six months prior to death.
 
-* Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+* Over the past `r as.character(as.english(year(end_date) - year(start_date)))` years, the percentage of time spent at home or in a community setting has gradually increased from `r summarise_data(basefile, include_years = "first") %>% pull(qom)`% in `r min(basefile$fy)` to `r latest$qom`% in `r latest$fy`.
 
-**Chart here**
 
-1. Figures for 2018/19 are incomplete and will be revised in October 2019 publication.
+<Div custom-style = "Chart Title">**Percentage of last six months of life spent at home or in a community setting;**</div>
+<Div custom-style = "Chart Title">**`r min(basefile$fy)` - `r max(basefile$fy)`**</div>
```

Comment:
Do we need to add a superscript "1" at the end, and have a note "1. Figures for 2018/19 are incomplete and will be revised in October 2019 publication."? I noticed you've got it in the May 2019 summary.